control/mpc_lateral_controller/src/mpc.cpp

```diff
@@ -38,7 +38,7 @@ MPC::MPC(rclcpp::Node & node)
 bool MPC::calculateMPC(
   const SteeringReport & current_steer, const Odometry & current_kinematics,
   AckermannLateralCommand & ctrl_cmd, Trajectory & predicted_trajectory,
-  Float32MultiArrayStamped & diagnostic)
+  Float32MultiArrayStamped & diagnostic, const bool is_driving_manually)
```

Comment:
Passing a value of `is_driving_manually` to the function that processes the MPC algorithm is not desirable from the perspective of function role definition. If the purpose is to overwrite the previous value, it would be better to handle it in the parent code:

```cpp
if (manual_driving) {
  m_mpc->setPreviousSteering(current_steering);
}
```

Reply:
fixed in 2156a9c9997d2e2511d18792cfdb30016896c7e4